acousticbrainz/data/user.py

```diff
@@ -0,0 +1,81 @@
+import psycopg2
+from flask import current_app
+from flask_login import UserMixin
+from werkzeug.exceptions import BadRequest, ServiceUnavailable
+
+
+def create(musicbrainz_id):
+    try:
+        connection = psycopg2.connect(current_app.config['PG_CONNECT'])
```

Comment:
how many places do you use this `connect` command? It feels to me like it should be an application-wide method than you can call. This way if we ever need to change arguments we only have to do it in one place